scripts/run-integration-tests.sh

```diff
@@ -7,23 +7,84 @@ cd "${PROJECT_ROOT}" || exit 1
 
 source "scripts/testlib.sh"
 check-set-swap-test-data-dir
-mkdir -p "${SWAP_TEST_DATA_DIR}/"{alice,bob,charlie}
 
-# Charlie uses deterministic ganache key #49
-CHARLIE_ETH_KEY="${SWAP_TEST_DATA_DIR}/charlie/eth.key"
-echo "87c546d6cb8ec705bea47e2ab40f42a768b1e5900686b0cecc68c0e8b74cd789" >"${CHARLIE_ETH_KEY}"
+RELAYER_PORT=7799
+
+# First 5 deterministic ganache keys (only 4 used at the moment)
+GANACHE_KEYS=(
+	"4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d" # Key 0
+	"6cbed15c793ce57650b9877cf6fa156fbef513c4e6134f022a85b1ffdd59b2a1" # Key 1
+	"6370fd033278c143179d81c5526140625662b8daa446c22ee2d73db3707e620c" # Key 2
+	"646f1ce2fdad0e6deeeb5c7e8e5543bdde65e86029e2fd9fc169899c440a7913" # Key 3
+	"add53f9a7e588d003326d1cbf9e4a43c061aadd9bc938c843a79e7b4fd2ad743" # Key 4
+)
```

Comment:
The first five keys in `ganache.go` are commented out and these are reserved for the integration tests and/or code that isn't in a `*_test.go` file.